### ISSUE TITLE
Feature request - Nearby device notification

### ISSUE THREAD

**Issue by Arturo2017** [tananaev/traccar#3073]
Now we have notifications of entry and exit in geofences.

Would be great to add to the device a field called for example "RADAR". Imagine the value of RADAR is 1000 mts, like a circle geofence. And then when the distance to any other Device-X is <1000 send notification "Device-X is on the radar".

It would be something like the new functionality of FaceBook "nearby friends"

**Comment by tananaev** [comment]
I remember someone already asked for a similar feature. Was it you?

**Comment by Arturo2017** [comment]
Looking at google _"traccar geofence around"_ I found the following: https://github.com/tananaev/traccar/issues/2308
```
Gar-za commented on 12 Sep 2016
A nice feature would be a to make a geofence around one of the units therefore making a moving geofence.
This would be ideal in a situation where you are delivering something to a work crew that moves all day.
```
And this one too:
https://github.com/vitalidze/traccar-web/issues/888

**Comment by tananaev** [comment]
Yes, I guess that's the comment I was thinking about. We will consider it for one of the future versions.

**Comment by Arturo2017** [comment]
Thanks Anton, it will be a very useful functionality especially when the AppTraccarManager gets to have PUSH notifications.

At the moment, I have achieved TraccarSoundAlerts on my smartphone using a Gmail account:

1- I created a filter of my emails received with the words "geofence + device"
2- Apply a label to that filter, for example "ALARM"
3- In Gmail Android App, assign a [SIREN.mp3](http://www.sonidosmp3gratis.com/sounds/POLICE.mp3) to that label

Now instantly, every time Traccar sends a geofence email notification, my smartphone rings the siren alarm!!!

**Comment by kirantpatil** [comment]
I also wanted to have [nearby devices api](https://github.com/traccar/traccar/issues/3218).

**Comment by kirantpatil** [comment]
Dear @tananaev,
 
Right now we don't have functionality to visualize the devices in the geofence as and when we create geofence.

How about having this feature as first step towards the Nearby device notification ?

Thanks.

**Comment by kirantpatil** [comment]
Dear @tananaev,

May I know how to proceed to implement this solution ?

If you have something in mind please let us know how to build it.

I would like to assign this feature to contribute from students. So that more people can start contribute this wonderful project.

Thanks.
